vllm/attention/backends/flashinfer.py

```diff
@@ -1,16 +1,10 @@
 from dataclasses import dataclass
 from typing import Any, Dict, List, Optional, Set, Tuple, Type
 
-try:
-    import flashinfer
-    from flash_attn import flash_attn_varlen_func
-    from flashinfer import BatchDecodeWithPagedKVCacheWrapper
-except ImportError:
-    flashinfer = None
-    flash_attn_varlen_func = None
-    BatchDecodeWithPagedKVCacheWrapper = None
-
+import flashinfer
```

Comment:
Does this import make `flashinfer` as the must-to-have dependency?

Reply:
Not really. The attention backends are lazily imported by `get_attn_backend`. Currently, the flashinfer backend is not imported unless the user specifies it via an env variable.

Reply:
I see. Thanks for the clarification :D